geoapps/io/DirectCurrent/constants.py

```diff
@@ -53,10 +53,10 @@
     "max_distance": np.inf,
     "horizontal_padding": 1000.0,
     "vertical_padding": 1000.0,
-    "window_center_x": 0.0,
-    "window_center_y": 0.0,
-    "window_width": 0.0,
-    "window_height": 0.0,
+    "window_center_x": None,
```

Comment:
These need to be 0.0 or else we get strings back from ui.json

Reply:
Done